src/optimum/nvidia/__init__.py

```diff
@@ -13,20 +13,14 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-DEFAULT_HF_HUB_TRT_REVISION: str = "trt-llm"
-
-DEFAULT_ENGINE_FOLDER = ".engine"
-TENSORRT_ENGINE_EXT = "engine"
-TENSORRT_TIMINGS_FILE = "timings.cache"
-OPTIMUM_NVIDIA_CONFIG_FILENAME = "build"
-OPTIMUM_NVIDIA_CONFIG_FILE = f"{OPTIMUM_NVIDIA_CONFIG_FILENAME}.json"
```

Comment:
This is moved to its dedicated file to avoid circular imports.

```
  File "/opt/optimum-nvidia-mine/src/optimum/nvidia/builder/base.py", line 41, in <module>
    from .. import OPTIMUM_NVIDIA_CONFIG_FILE, TENSORRT_TIMINGS_FILE
ImportError: cannot import name 'OPTIMUM_NVIDIA_CONFIG_FILE' from partially initialized module 'optimum.nvidia' (most likely due to a circular import) (/opt/optimum-nvidia-mine/src/optimum/nvidia/__init__.py)
```